src/utils/viewer/viewer.cpp

```diff
@@ -1048,6 +1078,25 @@ void Viewer::keyPressEvent(KeyEvent& event) {
     default:
       break;
   }
+
+  // Update map of moving/looking keys which are currently pressed
+  for (const auto& [k, b] : keysPressed) {
+    if (key == k) {
+      keysPressed[key] = true;
+    }
+  }
+
+  redraw();
+}
+
+void Viewer::keyReleaseEvent(KeyEvent& event) {
+  // Update map of moving/looking keys which are currently pressed
+  const auto key = event.key();
+  for (const auto& [k, b] : keysPressed) {
+    if (key == k) {
+      keysPressed[key] = false;
+    }
+  }
```

Comment:
you should use
`if (keyPressed.count(key) > 0)`
instead of the loop and if check here too.